src/index.mjs

```diff
@@ -52,20 +52,25 @@ function prefetcher(url) {
  * @param {function} options.timeoutFn - Custom timeout function
  */
 export default function (options) {
-  options = options || {};
+  options = Object.assign({
+    timeout: 2e3,
+    priority: false,
+    timeoutFn: requestIdleCallback,
+    el: document,
+  }, options);
 
-  observer.priority = !!options.priority;
+  observer.priority = options.priority;
```

Comment:
Did we mean to drop https://github.com/GoogleChromeLabs/quicklink/blob/c578c32c1c511f7cef7c1859fc480140f1e3ceb0/src/index.mjs#L62 as priority is back in the Object.assign block? :)

Reply:
This one writes into the Observer instance, which is referenced within `prefetcher()` – it's just a ploy to move the option value out of the function.

Here I just removed the `!!` which was used to cast undefined -> false.

---

Checking on `document.head` within Edge for ya

Reply:
Thanks! Appreciate you checking :)

Reply:
Fantastic. Thanks for verifying, Luke!